src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -99,7 +103,8 @@ public class ConsensusManager : IConsensusManager
             ConcurrentChain chain,
             IBlockPuller blockPuller,
             IBlockStore blockStore,
-            IInvalidBlockHashStore invalidHashesStore)
+            IInvalidBlockHashStore invalidHashesStore,
```

Comment:
What is `IInvalidBlockHashStore` hmm need to check it out.

Reply:
It's a list of banned block's hashes. 
We used to use it in consensusLoop and then forgot to add to rules engine.
I've wired it up a few days ago.